tests/unit/common/test_sensors.py

```diff
@@ -0,0 +1,202 @@
+from boat_simulator.common.sensors import WindSensor, GPS
+import numpy as np
+from boat_simulator.common.generators import (
+    MVGaussianGenerator,
+    ConstantGenerator,
+    GaussianGenerator,
+)
+import pytest
+
+
+@pytest.fixture
+def init_data():
+    return np.array([1, 0])
+
+
+class TestWindSensor:
+    def test_wind_sensor_init(self, init_data):
+        error_fn = None
+        ws = WindSensor(
+            wind=init_data,
+            wind_noisemaker=error_fn,
+        )
+
+        assert ws.wind_noisemaker == error_fn
+        assert np.all(ws.wind == init_data)
+
+    def test_wind_sensor_read_no_error(self, init_data):
+        error_fn = None
+        ws = WindSensor(
+            wind=init_data,
+            wind_noisemaker=error_fn,
+        )
```

Comment:
I think this might be implicitly what is done. I made the noisemaker type `Optional[GeneratorTypes...]` which technically makes it support the `None`. However, I have to double-check this statement...

Reply:
> Is it possible to just not specify the noise maker at all and have that essentially be equivalent to declaring it as None? Being able to either explicitly pass in None or not is advantageous.

After a lot of work playing around with dataclasses, this behaviour is now supported.